wordpress-mobile/WordPress-Android
# Issue/7463 post list settings gone on rotate

Partially fixes #7463 - this resolves the problem with the Post Status and Post Format dialogs disappearing upon rotation, a separate PR (#7542) tackles the Publish Date dialog disappearing.

To test:

- Open post settings
- Tap Status or Format
- Rotate the device
- Notice that the dialogs continue showing, and retain their current selection
